pkgs/development/python-modules/file-read-backwards/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, buildPythonPackage, fetchPypi, mock }:
+
+buildPythonPackage rec {
+  pname = "file-read-backwards";
+  version = "2.0.0";
+
+  src = fetchPypi {
+    pname = "file_read_backwards";
+    inherit version;
+    sha256 = "fd50d9089b412147ea3c6027e2ad905f977002db2918cf315d64eed23d6d6eb8";
+  };
+
+  checkInputs = [ mock ];
+  pythonImportsCheck = [ "mock" ];
```

Oh, `pythonImportsCheck` should verify that this package provides the expected modules. In this case, it should be:
```suggestion
  pythonImportsCheck = [ "file_read_backwards" ];
```